hikari/impl/stateless_event_manager.py

```diff
@@ -25,7 +25,6 @@
 
 __all__: typing.List[str] = ["StatelessEventManagerImpl"]
 
-import logging
```

Comment:
I assume you just haven't gotten around to this yet but do we still need the stateless vs stateful event manager?

Reply:
For now I have made it so it uses stateless event manager if the whole cache is disabled. Before merging into master I will see what I do. The thing that is worrying me rn and kept me from merging them is iterating throught stuff and making a call for that. If its disabled, they wont really make sense. As I type this, I realize I can check if the cache is enabled for what I want to do before doing it. But will think about it. For now, this pr will have them separate.